### PR TITLE
Add python3 support.

### PR DESCRIPTION
**Pull request by cbalint13**
This PR adds support for python3.

### CODE REVIEW

File: wincnn.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 from sympy import symbols, Matrix, Poly, zeros, eye, Indexed, simplify, IndexedBase, init_printing, pprint
 from operator import mul
+from functools import reduce
```
Comment (andravin) [inline wincnn.py:4]: We should also add this import to maintain python2 compatibility:

`from __future__ import print_function`

**Review by andravin — CHANGES_REQUESTED** [review]
Thanks for the pull request.

Looks good .. just need to add the print_tools import, and then I will test on python2 and python3.

File: wincnn.py (hunk 1 above)
Comment (cbalint13) [reply wincnn.py:4]: done

**Comment by andravin** [comment]
Works for me, will merge. Thanks!

**Pull request merged**